WikibaseDataModel.php

```diff
@@ -7,7 +7,10 @@
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
  */
 
-define( 'WIKIBASE_DATAMODEL_VERSION', '5.0.0 alpha' );
+/**
+ * @deprecated since 5.0
```

Comment:
Why is this actually deprecated in here? If I understand Katie's comment correctly we want to keep this in here and not remove it anytime soon.